### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by MauriceHendrix**
## Description

@MichaelClerx asekd for some changes to the pullrequest template in issue #164. Here are the proposed changes. I generally agree however I would like something in there about having checked docs & typos as I forget this all the time :)

## Motivation and Context

We're trying to improve our working practices so we can get more & better work done! Discussion on #164 and previous (closed) pr #163. 

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
(More of a style/preference issue than bug)

## Checklist:

- [X] I have updated all documentation necessary.
- [ ] I have checked spelling in (new) comments.
(No updates necessary)

##Testing
- [ ] Testing is done automaticly and codecov shows test coverage
- [X] This cannot be tested automatically 
This is a pull request template. This PR **IS the test**

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -6,9 +6,8 @@
 <!--- If it fixes an open issue, please link to the issue here, using the 'fixes #<issue>' syntax. -->
 
 ## How Has This Been Tested?
-<!--- Please describe in detail how you tested your changes. -->
-<!--- Include details of your testing environment, and the tests you ran to -->
-<!--- see how your change affects other areas of the code, etc. -->
+- [ ] New automated tests added
```
Comment (MichaelClerx) [inline pull_request_template.md]: Better, but doesn't the coverage testing take care of this? I'm not sure what the added value of this bit is

File: pull_request_template.md [hunk 2]
```diff
@@ -18,10 +17,6 @@
...
 <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
-<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
-- [ ] My code follows the code style of this project.
-- [ ] My change requires a change to the documentation.
-- [ ] I have updated the documentation accordingly.
-- [ ] I have read the **CONTRIBUTING** document.
-- [ ] I have added tests to cover my changes.
-- [ ] All new and existing tests passed.
+- [ ] I have updated all dcoumentation necessary.
+- [ ] I have check spelling in (new) comments.
```
Comment (MichaelClerx) [inline pull_request_template.md]: Nice!

(and checked :D )

File: pull_request_template.md [hunk 3]
```diff
@@ -18,10 +17,6 @@
...
 ## Checklist:
 <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
-<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
-- [ ] My code follows the code style of this project.
-- [ ] My change requires a change to the documentation.
-- [ ] I have updated the documentation accordingly.
-- [ ] I have read the **CONTRIBUTING** document.
-- [ ] I have added tests to cover my changes.
-- [ ] All new and existing tests passed.
+- [ ] I have updated all dcoumentation necessary.
```
Comment (MichaelClerx) [inline pull_request_template.md]: documentation

File: pull_request_template.md (hunk 2 above)
Comment (MauriceHendrix) [reply pull_request_template.md]: case in point 🥇

File: pull_request_template.md (hunk 1 above)
Comment (MauriceHendrix) [reply pull_request_template.md]: Sure we could take it out it's more when adding things you can't test automatically like PR templates

File: pull_request_template.md [hunk 4]
```diff
@@ -18,10 +13,10 @@
...
-- [ ] My change requires a change to the documentation.
-- [ ] I have updated the documentation accordingly.
-- [ ] I have read the **CONTRIBUTING** document.
-- [ ] I have added tests to cover my changes.
-- [ ] All new and existing tests passed.
+- [ ] I have updated all documentation necessary.
+- [ ] I have checked spelling in (new) comments.
+
+##Testing
+- [X] Testing is done automaticly and codecov shows test coverage
```
Comment (jonc125) [inline pull_request_template.md]: Speeling ;)

Comment (MauriceHendrix) [reply pull_request_template.md]: ai ai ai

Comment (jonc125) [reply pull_request_template.md]: automaticly -> automatically

**Pull request merged**